.github/workflows/testing.yml

```diff
@@ -31,58 +31,97 @@ jobs:
           - 27017:27017
 
     runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash -l {0} # enables conda/mamba env activation by reading bash profile
     strategy:
       matrix:
         python-version: ["3.9", "3.10", "3.11"]
 
     steps:
-      - uses: actions/checkout@v4
+      - name: Check out repo
+        uses: actions/checkout@v4
 
-      - uses: actions/setup-python@v5
-        with:
-          python-version: ${{ matrix.python-version }}
+      - name: Set up micromamba
+        uses: mamba-org/setup-micromamba@main
 
-      - name: Install enumlib
+      - name: Create mamba environment
         run: |
-          cd ..
-          git clone --recursive https://github.com/msg-byu/enumlib.git
-          cd enumlib/symlib/src
-          export F90=gfortran
-          make
-          cd ../../src
-          make enum.x
-          sudo mv enum.x /usr/local/bin/
-          cd ..
-          sudo cp aux_src/makeStr.py /usr/local/bin/
-        continue-on-error: true # This is not critical to succeed.
+          micromamba create -n a2 python=${{ matrix.python-version }} --yes
+
+      - name: Install uv
+        run: micromamba run -n a2 pip install uv
+
+      - name: Install conda dependencies
+        run: |
+          micromamba install -n a2 -c conda-forge enumlib packmol bader openbabel openff-toolkit==0.16.2 openff-interchange==0.3.22 --yes
 
       - name: Install dependencies
         run: |
+          micromamba activate a2
           python -m pip install --upgrade pip
           mkdir -p ~/.abinit/pseudos
           cp -r tests/test_data/abinit/pseudos/ONCVPSP-PBE-SR-PDv0.4 ~/.abinit/pseudos
-          pip install .[strict,tests,abinit]
-          pip install torch-runstats
-          pip install --no-deps nequip==0.5.6
+          uv pip install .[strict,tests,abinit,classical_md]
```

Comment:
No need to put `classical_md` here if your dependencies are already included in the `strict` set of requirements (I should look at copying the abinit dependencies into strict too).

Reply:
Removed